posts/Yoann/2018-02-07-hibernate-search-5-9-0-Final.adoc

```diff
@@ -0,0 +1,97 @@
+= Hibernate Search 5.9.0.Final was just released
+Yoann Rodière
+:awestruct-tags: [ "Hibernate Search", "Elasticsearch", "Releases" ]
+:awestruct-layout: blog-post
+---
+
+We just published 5.9.0.Final, the first stable release in the 5.9 branch.
+This release brings a brand-new JSR-352 integration for mass indexing,
+WildFly feature packs, and better integration to modular environment.
+
+== What's new?
+
+If you tried out the Beta/CR, know that this release is virtually identical to 5.9.0.CR1,
+with only a few changes to our internal build process and a micro update to the Elasticsearch client.
+
+If you didn't, then there is something new for you!
+
+[[jsr352]]
+=== JSR 352 (Batch for Java) mass indexing job
+
+A JSR 352 mass indexing job was added, adding in particular the ability to restart
+from a checkpoint when indexing fails.
+
+See https://docs.jboss.org/hibernate/search/5.9/reference/en-US/html_single/#jsr352-integration[the documentation]
+for more information,
+and http://in.relation.to/2017/11/28/introducing-hibernate-search-jsr352-mass-indexing-job/[this blog post]
+for an introduction to JSR 352 in Hibernate Search.
+
+[[wildfly-feature-packs]]
+=== JBoss modules now distributed as feature packs
+
+The Hibernate Search JBoss Modules, for use in WildFly or JBoss EAP,
+are now distributed as Wildfly feature packs, which makes provisioning
+a WildFly server with an upgraded Hibernate Search version easier than ever.
+The feature packs can also be consumed using http://wildfly-swarm.io/[WildFly Swarm].
+
+Also, the modules are now distributed as *multiple* feature packs,
+which means for example that you won't need to add the Elasticsearch clients
```

Comment:
before getting to the "for example" part we probably should state what it means.

Suggestion:

    which means that components are split in fine grained packages, allowing you to pick only the parts you will need. For example..